### DISCUSSION TITLE
GOIP-4

### DISCUSSION THREAD

**Discussion by DoctorMOMcv** [mikopbx/Core#660] [Q&A]
По возможности опубликуйте настройку IP PBX и GOIP-4 для совместной работы. На wiki нет иніформации, другие ссілки не работают.

**Comment by boffart** [comment]
Инструкции есть на старой wiki
https://wiki.mikopbx.ru/faq#podkljuchenie_shljuzov

**Reply by DoctorMOMcv** [reply]
Не буду говорить причину, умный поймет, даже с VPN нет доступа к этой странице.
